apps/api/src/routers/demo.py

```diff
@@ -15,3 +17,18 @@ async def me(username: Annotated[Union[str, None], Cookie()] = None) -> str:
 async def square(value: Annotated[int, Form()]) -> int:
     """Calculate the square of the value."""
     return value * value
+
+
+@router.post("/user")
+async def get_user(name: Annotated[str, Form()]) -> list[dict[str, Any]]:
+    results = await retrieve(Collection.USERS, {"name": name}, ["name", "ucinetid"])
+    for i in range(len(results)):
+        results[i]["_id"] = str(results[i]["_id"])
```

Comment:
Nitpick [non-blocking since just for demo]: we could use direct object iteration instead of range-based iteration.
```suggestion
    for result in results:
        result["_id"] = str(result["_id"])
```